modules/scripts/genindex.py

```diff
@@ -96,6 +96,11 @@ def get_index(root, subprojects, sub_docpaths):
 
     return '\n\n'.join(index) + '\n'
 
+def add_javadoc(content):
```

Comment:
I don't think this is needed. But if you still wanna implement this, don't directly access environment vars. Instead pass a parameter for consistency.